client/src/components/AccordionTabContent.jsx

```diff
@@ -61,7 +99,20 @@ const AccordionTabContent = (props) => {
       //designedFor               -->     office/travel/commute
       //Fabric                    -->     description (fabric)
       //Fit (relaxed, hip length) -->     description (fit)
-      <div>╮ (. ❛ ᴗ ❛.) ╭ {placeholderFeatures} </div>
+      <ItemFeaturesContainer>
+        <ItemFeature>
+          <ItemFeatureName>Designed For</ItemFeatureName>
+          {props.features.designedFor[0]}
+        </ItemFeature>
+        <ItemFeature>
+          <ItemFeatureName>
+            {props.features.fit.join(', ').toUpperCase()}
+          {/* {props.features.fit.map((fit) => {
+            return fit
+          })} */}
+          </ItemFeatureName>
+        </ItemFeature>
+      </ItemFeaturesContainer>
```

Comment:
This all looks good to me, nice looking components.

Reply:
Thanks Garrett!